ReactWindows/ReactNative/Views/WebView/Events/WebViewLoadingFinishEvent.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.UIManager.Events;
+using System;
+
+namespace ReactNative.Views.WebView.Events
+{
+    class WebViewLoadingFinishEvent : Event
+    {
+        private readonly string _url;
+        private readonly bool _loading;
+        private readonly string _title;
+        private readonly bool _canGoBack;
+        private readonly bool _canGoForward;
+
+        public WebViewLoadingFinishEvent(int viewTag, string url, bool loading, string title, bool canGoBack, bool canGoForward)
```

Comment:
Can we use a single event type for LoadingFinish and LoadingStart and just parameterize the event name?  Might also make sense to put each param on a different newline.